### PR TITLE
Add menu 

### PR DESCRIPTION
**Pull request by Abdo9826**
Add HTML-CSS and JavaScript linters
Add CSS style 
Add Java script file

### CODE REVIEW

File: a.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const humberger = document.querySelector('.humberger');
+const ulnav = document.querySelector('.ulnav');
+const logo = document.querySelector('.aa');
+
+humberger.addEventListener('click', () => {
+  logo.classList.toggle('none');
+  ulnav.classList.toggle('active');
+  humberger.classList.toggle('active');
+});
```
Comment (Wes-Isaac) [inline a.js:12]: - [ ] One of the project requirements is when one of the links is clicked from the menu it should go to the appropriate part of the page which you implemented nicely but the mobile menu should disappear which you didn't, Kindly fix that.

image

File: a.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const humberger = document.querySelector('.humberger');
+const ulnav = document.querySelector('.ulnav');
```
Comment (Wes-Isaac) [inline a.js:2]: - [ ] [_OPTIONAL_] It's better to give meaningful file names, instead of a.js better to make it main.js or index.js.

File: a.css [hunk 3]
```diff
@@ -708,14 +734,19 @@ textarea {
...
+    left: 0;
+    padding: 0;
+    width: 0;
+    height: 0;
     display: block;
     display: flex;
     list-style-type: none;
-    margin-top: 26px;
+    margin-top: -23px;
     margin-left: 860px;
```
Comment (Wes-Isaac) [inline a.css:752]: - [ ] The style in the design is a little different from the one you implemented. Kindly fix it.

### Your Implementation
image

### Figma's Design
image

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @Abdo9826, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linters are on point ✔️.
- Readme is professional 👀.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: a.js (hunk 1 above)
Comment (KDlamini) [inline a.js:12]: Kindly add the functionality that will make the menu disappear when a menu item is clicked as stated by the previous reviewer. Currently, when I scroll up after clicking on a menu item, the mobile menu is still active. 

- The same functionality that closes the menu when the times(X) icon is clicked should also be applied when any of the menu list items are clicked.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @Abdo9826 🙋‍♀️,  

You have done an incredible job implementing some of the changes requested by the previous reviewer. 👏🏾 
However, there are still some changes that aren't addressed yet. You are almost there! 💪🏾 

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**